x/crisis/keeper/msg_server.go

```diff
@@ -13,11 +13,11 @@ func (k Keeper) VerifyInvariant(goCtx context.Context, msg *types.MsgVerifyInvar
 	ctx := sdk.UnwrapSDKContext(goCtx)
 	constantFee := sdk.NewCoins(k.GetConstantFee(ctx))
 
-	sender, err := sdk.AccAddressFromBech32(msg.Sender)
+	err := sdk.ValidateAccAddress(msg.Sender)
```

Comment:
Do we need validation here? Seems `msg.validate()` has already been called before here.